RecipeBananabread.prefab

```diff
@@ -0,0 +1,17 @@
+{
+	    "ProcessDefinition": {
+	        "processType": "Cooking:Cooking"
+	    },
+	    "CraftingStationRecipe": {
+	        "recipeId": "Hot food|Cooking:Bananabread�,
+
+
+	        "recipeComponents": ["1*Cooking:flour","1*Cooking:sugar�,�3*Cooking:banana�,�1*Cooking:chocolate�],
```

Comment:
Make sure all components are existing food items.
flour does not exist. CornFlour does